tests/phpunit/elementor/core/kits/documents/kits/test-kit.php

```diff
@@ -43,9 +43,12 @@ public function test_save__kits_revision_ensure_changed() {
 
 	public function test_save__kits_revision_ensure_same() {
 		// Arrange.
+		$initial_count = count( Revisions_Manager::get_revisions( $this->kit->get_main_id() ) );
+
 		$this->kit->save( [ 'settings' => $this->kit->get_settings() ] );
 
 		$excepted_count = count( Revisions_Manager::get_revisions( $this->kit->get_main_id() ) );
+		$excepted_count += $initial_count;
```

Comment:
How can it be? Only 1 as been added

Reply:
`$initial_count` is 1, probably it is creating the first revision on bootstrap (wp load)... i remember that i didn't find solution for this, do you wish i'll debug it again?

Reply:
Please use hardcoded values e.g.`$excepted_count = 1` in order to make it clear

Reply:
Handled.